UIP.py

```diff
@@ -1,12 +1,8 @@
 from scheduler import scheduler
 
-print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
-
 if __name__ == "__main__":
-<<<<<<< 2185a6f5ba43b272c70dd78f1edf0696934debb4
-  print("Press 1 to connect to internet or any other key to use this application in offline mode")
-  offline = int(input()) != 1
-  scheduler(offline)
-=======
-	scheduler()
->>>>>>> Added a nice message For UIP to show on the main screen
+      print ("Hey this is UIP! you can use it to download images from reddit 	and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")
+      offline = int(input())!=1
+      scheduler(offline)
+      scheduler()

```

Comment:
we dont need this remove this please, also always test code by running `python3 UIP.py --force` before push :P